blocks/library/pullquote/index.js

```diff
@@ -109,8 +95,8 @@ export const settings = {
 						onFocus={ onSetActiveEditable( 'cite' ) }
 					/>
 				) }
-			</blockquote>,
-		];
+			</blockquote>
+		);
 	} ),
 
 	save( { attributes } ) {
```

Comment:
The `save` implementation should no longer be responsible for applying the `align` class.

Same goes for all other refactored blocks.

Reply:
I missed that, good catch 👍